### ISSUE TITLE
Wiki Upload Request: German and English Wikipedia 2017-11

### ISSUE THREAD

**Issue by Ope30** [gnosygnu/xowa#179]
Heyo. I think we shouldn't hesitate ignoring the "Refresh Issue" for a while, atleast in my point of view. It might be fixed in a next release, or build. Who knows.

Other than that, what do you think about a new release regarding German-English Wikipedia? Of course, if there's nothing between your schedule. Otherwise, this can wait. But I wouldn't mind downloading a new release after a while. :-3

Concerning Commons: Let's just put the issue aside. You mentioned Categories or someting like that. That would be grandiose if you would add them! Big fan of Commons.

How does it look for you?!

**Comment by gnosygnu** [comment]
Yup. I kicked off the 2017-11 builds now. English Wikipedia won't be ready until sometime next week, and German Wikipedia sometime afterwards

Regarding Commons, I looked at the code yesterday, and couldn't find a possible cause. As said, I'll add some more logging, though that will come later this week.

Thanks.

**Comment by Ope30** [comment]
Sorry for asking this, if you're ever gonna add them, how big are the Categories going to be? Aren't they over like 30GB or something? I think Commons is pretty big.

**Comment by gnosygnu** [comment]
Commons Categories would probably be about 20 GB. The compressed file is 4.7 GB: https://dumps.wikimedia.org/commonswiki/20171103/commonswiki-20171103-categorylinks.sql.gz

I have the Commons category build on my list. I'll see if I can get them done for 2017-11

Thanks.

**Comment by Ope30** [comment]
> Commons Categories would probably be about 20 GB. The compressed file is 4.7 GB: https://dumps.wikimedia.org/commonswiki/20171103/commonswiki-20171103-categorylinks.sql.gz

> I have the Commons category build on my list. I'll see if I can get them done for 2017-11

👍

**Comment by Ope30** [comment]
> I have the Commons category build on my list. I'll see if I can get them done for 2017-11

Oh please! 👍  I've been waiting for a long time. :(
